.circleci/config.yml

```diff
@@ -24,6 +16,14 @@
 
 version: 2.1
 
+orbs:
```

Comment:
do you have the channel set up for this?

Reply:
yep #aztec3-ci (same as aztec-packages)